picard/ui/scripteditor.py

```diff
@@ -460,9 +460,12 @@ def toggle_wordwrap(self):
     def import_script(self):
         """Import the current script from an external text file.
         """
+        default_path = os.path.normpath(QtCore.QStandardPaths.writableLocation(QtCore.QStandardPaths.DocumentsLocation))
```

Comment:
Perhaps we should make those properties of the class:

```python
default_script_directory = os.path.normpath(QtCore.QStandardPaths.writableLocation(QtCore.QStandardPaths.DocumentsLocation))
default_script_filename = "picard_naming_script.pts"
```

So it can be overriden by subclassing, and reduce code redundancy.